openapi_python_client/__init__.py

```diff
@@ -9,7 +9,6 @@
 from typing import Any, Dict, Optional, Sequence
 
 import httpx
-import yaml
```

Comment:
On main yaml is still needed. Can you rebase this branch so checks will still run?